lib/tpm_hash.c

```diff
@@ -14,8 +14,8 @@ UINT32 tpm_hash(TSS2_SYS_CONTEXT *sapi_context, TPMI_ALG_HASH hashAlg,
 
 static TPM_RC hash_sequence_ex(TSS2_SYS_CONTEXT *sapi_context,
 
-    TPMI_ALG_HASH hashAlg, UINT32 numBuffers, TPM2B_MAX_BUFFER *bufferList,
-    TPM2B_DIGEST *result) {
+TPMI_ALG_HASH hashAlg, UINT32 numBuffers, TPM2B_MAX_BUFFER *bufferList,
+        TPM2B_DIGEST *result) {
```

Comment:
There's quite a few superfolous formatting changes, normally, when you want to get a patch in, make it the bare minimum to do what you need and submit these types of changes as a separate patch.

I am going to let it slide this time, but ill start enforcing it on subsequent reviews. So if other things as I read down cause me to reject your changes, the next update to the PR should have changes like these split out.